### DISCUSSION TITLE
/usr/bin/nerdctl _NERDCTL_INTERNAL_LOGGING process consumes lots of file description

### DISCUSSION THREAD

**Discussion by Somiacao** [containerd/nerdctl#2440] [Q&A]
hello, there is a question about process _NERDCTL_INTERNAL_LOGGING
from my searching, the process is used to pull logs from containers.
but the process will consume lots of fd.
if I deploy 700 containers in a machine, the process will consume 23w+ fd, contanerd-shim consume 38w+ fd

**Comment by fahedouch** [comment]
duplicate https://github.com/containerd/nerdctl/issues/2441
